DecipherNow/gm-fabric-dashboard
# fix: changed ID to Name in instance view

Resolves #1024 

## Proposed Changes

  - Swapped hardcoded 'ID' for 'Name'
  
## How to Test

  - Navigate to instance view and make sure table heading for left most column says 'Name'
